addon/-private/formatters/-base.ts

```diff
@@ -29,89 +24,23 @@ export interface BaseOptions {
 // eslint-disable-next-line @typescript-eslint/ban-types
 export default abstract class FormatterBase<KnownOptions extends {}> {
   protected readonly config: FormatterConfig;
-  protected readonly readFormatConfig: () => Formats;
-
-  static type: keyof Formats;
-
-  protected abstract readonly createNativeFormatter: (
-    locales: string | string[],
-    options: KnownOptions
-  ) => Intl.DateTimeFormat | Intl.RelativeTimeFormat | Intl.NumberFormat | IntlMessageFormat;
+  protected readonly getIntl: (locale: string | string[]) => IntlShape<string>;
+  static type: keyof Formats | 'message';
```

Comment:
Curious why the `| 'message'` change here

Reply:
`keyof Formats` only contains shorthand formattable types like `date`/`time`/`number`/`relative`. `message` has no shorthand format so that's why it's `| 'message'`